gui/include/avopenglwidget.h

```diff
@@ -28,15 +28,36 @@ extern "C"
 #include <libavcodec/avcodec.h>
 }
 
+#define MAX_PANES 3
+
 class VideoDecoder;
 class AVOpenGLFrameUploader;
 
+struct PlaneConfig
+{
+	unsigned int width_divider;
+	unsigned int height_divider;
```

Comment:
Silent assumption that most of formats will have additional panes with resolution equal or less than original frame.